src/components/ui/Checkbox.js

```diff
@@ -1,14 +1,19 @@
 import React from 'react';
 import PropTypes from 'prop-types';
+import { injectIntl } from 'react-intl';
 
-function Checkbox({ id, callback, label, defaultChecked}) {
+function Checkbox({ id, callback, label, defaultChecked, checked , labelId, intl }) {
   return (
     <div className="c-checkbox">
       <div className="checkbox">
-        <input type="checkbox" id={id} onChange={callback} defaultChecked={defaultChecked}/>
+        { typeof checked !== 'undefined' ? 
```

Comment:
We don't want checked to be false if the checked is not defined. We don't want it to have the param. So is still uncontrolled.